src/styles/GlobalStyle.js

```diff
@@ -3,8 +3,20 @@ import reset from 'styled-reset';
 
 const GlobalStyle = createGlobalStyle`
   ${reset}
-	
-	// 전역스타일
+
+  *{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+  body{
+    font-family: 'Noto Sans KR', sans-serif;
+  }
+	a{
+    text-decoration: none;
+    color: inherit;
+  }
```

Comment:
전역에 스타일링을 하는 경우 꼭 팀원들하고도 공유해주세요!
```suggestion
  *{
    box-sizing: border-box;
    margin: 0;
    padding: 0;
  }
  
  body{
    font-family: 'Noto Sans KR', sans-serif;
  }
  
  a{
    text-decoration: none;
    color: inherit;
  }
```
그리고 위와 같이 GlobalStyle.js에서는 프리티어가 작동하지 않으므로 직접 줄바꿈과 들여쓰기를 해서 가독성을 높여주세요 :)

Reply:
넵 줄바꿈과 들여쓰기 완료하고 팀원들에게 전달했습니다!!